README.md

```diff
@@ -1,5 +1,74 @@
-# The National Archivs cookie banner consent
+# Cookie banner consent Wordpress plugin
 
-v1.0 -- 2021-03-30
+- [Project findings](https://github.com/nationalarchives/ds-cookie-consent/blob/mihai/wp_plugin/docs/about-the-project.md)
 
-- [Project findings](https://github.com/nationalarchives/ds-cookie-consent/blob/main/docs/about-the-project.md)
+## Installation
+- Upload, install and activate the plugin
+
+## Create Cookies Settings page
+
+- Add shortcode [ds-cookie-consent-settings] to the website
+
+- Content for the cookie sections ca be added from TNA Cookie Consent dashboard settings page
+
+## Banner
+<img width="1263" alt="Screenshot 2021-04-20 at 10 35 43" src="https://user-images.githubusercontent.com/5245264/115373917-3b91b980-a1c4-11eb-82f7-96acab4a45cb.png">
+
+
+### HTML
+
+[https://github.com/nationalarchives/ds-cookie-consent/blob/develop/includes/utils.php#L20-L31](Banner)
+
+### JavaScript
+
+[https://github.com/nationalarchives/ds-cookie-consent/blob/develop/src/api/dsCookieConsentBannerAPI.js](API)
```

Comment:
Not sure if this is expected but I'm getting a 404 for this page.

Reply:
Done